examples/existing-image/variables.tf

```
@@ -1,74 +1,68 @@
-# General
-# -------
-variable "acm_certificate_arn" {
+variable "ami_id" {
   type        = string
-  description = "ACM certificate ARN to use with load balancer"
+  description = "AMI ID of the custom image to use for TFE instances. If this value is provided, you do not need any of the following ami variable values."
```

When you're in VSCode, try and keep an eye on how long the lines are getting so that they're uniform.
```suggestion
  description = <<DESC
  AMI ID of the custom image to use for TFE instances. If this value is provided, you do not need any of the following 
  ami variable values.
  DESC
```